enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/IverksettDto.kt

```diff
@@ -76,6 +80,18 @@ data class VedtaksperiodeDto(
         val periodeType: VedtaksperiodeType
 )
 
+data class FeilutbetalingDto(
+        val tilbakekrevingsvalg: Tilbakekrevingsvalg,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL
+        val varseltekst: String?,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL.
+        // Hvis det er påkrevd og mangler, så vil det hentes fra simulering
+        val sumFeilutbetaling: BigDecimal?,
+        // Kreves bare hvis tilbakekrevingsvalg = OPPRETT_TILBAKEKREVING_MED_VARSEL.
+        // Hvis det er påkrevd og mangler, så vil det hentes fra simulering
+        val perioder: List<Periode>?
```

Comment:
Vil vi sette sum og perioder manuelt? 🤔

Reply:
Ja, ser sånn ut. Kallet til `familie-tilbake` krever følgende innhold i `Varsel`:

- varseltekst
- sumFeilutbetaling
- perioder

Reply:
Vi får (dvs avleder den informasjonen) fra simuleringen i ef-sak, ser det ut til.

Reply:
Men kan finne info'en når vi iverksetter, også.

Reply:
Ok, da må vi finne noen "triks i ludo" for å utlede disse periodene.